optimus/wdl_tasks/FastqToUBam.wdl

```diff
@@ -0,0 +1,28 @@
+
+task FastqToUBam {
+  File gz_fastq_file  # input fastq file
+  String sample_name  # name of sample matching this file, inserted into read group header
+
+  # estimate that bam is approximately equal in size to fastq, add 20% buffer
+  Int estimated_disk_required = ceil(size(gz_fastq_file, "G") * 2.2)
```

Comment:
In general I think we should be very generous with resource allocation, then benchmark and optimize where needed. For production pipelines, not having enough wiggle room will cause lots of headaches with failed workflows, even if it only happens for 1 in 1000 bundles. If this is a very well behaved, predictable task in terms of disk requirements then 20% is probably fine. If we're less sure, then I would err more on the side of being too generous and give, say 2x as much as we typically need.

Reply:
Message received: in the general case, err on the side of running the task without problems instead of over-optimizing to save money, but inducing operational problems. 

In this case the majority of the information in the two file types is text data (read name, sequence, and quality) which is present in both file types, and both types us gzip compression, so the types are quite consistently similar in size. I could boost this up a little bit, but 2x is definitely not required for this particular task.